packages/office-ui-fabric-react/src/Modal.ts

```diff
@@ -1,2 +1 @@
 export * from './components/Modal/index';
-export { Modal as default } from './components/Modal/index';
```

Comment:
why remove this line? This would be a breaking change for people already using it..

Reply:
Reverted thx

---
In reply to: [207694831](https://github.com/OfficeDev/office-ui-fabric-react/pull/5802#discussion_r207694831) [](ancestors = 207694831)